openpype/hosts/maya/api/lib_renderproducts.py

```diff
@@ -383,7 +393,7 @@ def get_files(self, product, camera):
             self.layer_data,
             force_aov_name=product.productName,
             force_ext=product.ext,
-            force_cameras=[camera]
+            force_cameras=[product.camera] or None
```

Comment:
The `or None` part will never be touched I think, since that list always have a member. No ?

Reply:
true. I don't see it ever reaching None